congestion-control-charter.txt

```diff
@@ -67,10 +70,9 @@ signals, including specification of explicit signals.
 * Progression of existing Informational or Experimental RFCs to higher maturity,
 if they meet the criteria.
 
-* Operational guidance, in general, including advice to upper layers such as regarding
-counterproductive application retransmissions and multiple connections to the
-same host.
-
+* Operational guidance, in general, including advice to upper layers or new
+transports. Examples include minimum frequency of feedback, counterproductive

```

Comment:
What is a new transport? it this something in lower layer like L3 or L2 transport or a new L4 transport protocol?

Reply:
L4

Reply:
lets make it clear then - s/transports/transport protocols.